.circleci/build_and_push_image.sh

```diff
@@ -11,7 +11,6 @@ then
     CIRCLE_SHA1=$(git rev-parse HEAD)
 fi
 
-apt-get install -y make jq
```

Comment:
Moved this installation to `.circleci/config.yml` so we could run this script without root access.